content/source/docs/enterprise/private/encryption-password.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Encryption Password"
+sidebar_current: "docs-enterprise2-private-encryption-password"
+---
+
+# Private Terraform Enterprise Encryption Password
+
+During the installation of Private Terraform Enterprise,
```

Comment:
This is awfully short! I think the standard is 80. But it is consistent so I'm fine with it. :)